lib/webtaskCreator.js

```diff
@@ -43,8 +44,15 @@ function createWebtaskCreator(args, options) {
                     merge: args.merge,
                     parse: args.parse,
                     secrets: args.secrets,
-                    params: args.params
+                    params: args.params,
+                    host: args.host
                 });
+            })
+            .catch(function (e) { 
+                return e && e.statusCode === 403; 
+            }, function (e) { 
+                console.log(Chalk.red('Error creating webtask: ' + e.message));
```

Comment:
All handlers are expected to return `Promise`s. Yes, apocalypse now. The intended way to get the behaviour you are looking for is to to add:

```js
const Errors = require('./errors')